### PR TITLE
Add helm chart and skaffold.yaml

### PR DESCRIPTION
**Pull request by fabubaker**

### CODE REVIEW

File: helm/rayyan-dedup/values.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    envFrom:
+      secretRefs:
+        - postgres
+      configMapRefs:
+        - common
+    resources:
+      requests:
+        cpu: 400m
+        memory: 2.7G
+    tolerations: &WORKER_TOLERATIONS
```
Comment (hammady) [inline helm/rayyan-dedup/values.yaml:20]: Tolerations are now part of rayyanlib templates.

File: README.md [hunk 2]
```diff
@@ -3,28 +3,19 @@
 # rayyan-dedup
```
Comment (hammady) [inline README.md:3]: Please insert a build badge here: create the pipeline, run and test it on the branch, then from the drop down get its badge markdown after setting branch to mater and scope to the Build stage.

**Review by hammady — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (fabubaker) [reply README.md:3]: Done

File: helm/rayyan-dedup/values.yaml (hunk 1 above)
Comment (fabubaker) [reply helm/rayyan-dedup/values.yaml]: Removed `WORKER_TOLERATIONS`

File: README.md (hunk 2 above)
Comment (hammady) [reply README.md:3]: To be consistent, I am trying to rename the pipeline to just `rayyan-dedup` then we have to update the badge. The rename is not working for some reason.

Comment (hammady) [reply README.md:3]: Also you didn't scope it to the build stage!

File: helm/rayyan-dedup/values.yaml [hunk 3]
```diff
@@ -17,7 +17,7 @@ workers:
       requests:
         cpu: 400m
         memory: 2.7G
-    tolerations: &WORKER_TOLERATIONS
+    tolerations:
```
Comment (hammady) [inline helm/rayyan-dedup/values.yaml:20]: Remove the tolerations block completely!

Comment (fabubaker) [reply helm/rayyan-dedup/values.yaml:20]: Wait, I don't understand, where else would you specify tolerations?

File: README.md (hunk 2 above)
Comment (fabubaker) [reply README.md:3]: There was no option to scope to the build stage!

Comment (fabubaker) [reply README.md:3]: I manually edited the URL and set it to Build.

Comment (fabubaker) [reply README.md:3]: > To be consistent, I am trying to rename the pipeline to just `rayyan-dedup` then we have to update the badge. The rename is not working for some reason.

I renamed the pipeline and updated the badge link.

Comment (hammady) [reply README.md:3]: I am not sure why no more scopes appear, same for the new scaltainer pipeline as well. I checked rayyancore pipeline though and it has all scopes. But observe the badge itself it says "never built"!

image

**Review by hammady — APPROVED** [review]

**Pull request merged**